### PR TITLE
Make test of Kmeans very very unlikely to fail

### PR DESCRIPTION
**Pull request by oeddyo**
In the test for Kmeans, simply put the trailing random numbers to be really really slow. 

Experiment result confirmed that, with random number to be 

rnd/dim, 100 tests it would fail once

With my change I ran 1000 tests, and it's still green.

### CODE REVIEW

**Comment by oeddyo** [comment]
https://github.com/twitter/scalding/issues/998

File: scalding-serialization-macros/src/test/scala/com/twitter/scalding/serialization/macros/MacroOrderingProperties.scala [hunk 1]
```diff
@@ -16,14 +16,14 @@ limitations under the License.
...
-import java.io.{ByteArrayOutputStream, InputStream}
+import java.io.{ ByteArrayOutputStream, InputStream }
 import java.nio.ByteBuffer
 
-import com.twitter.scalding.serialization.{JavaStreamEnrichments, Law, Law1, Law2, Law3, OrderedSerialization, Serialization}
-import org.scalacheck.Arbitrary.{arbitrary => arb}
-import org.scalacheck.{Arbitrary, Gen, Prop}
-import org.scalatest.prop.{Checkers, PropertyChecks}
-import org.scalatest.{FunSuite, ShouldMatchers}
+import com.twitter.scalding.serialization.{ JavaStreamEnrichments, Law, Law1, Law2, Law3, OrderedSerialization, Serialization }
```
Comment (ianoc) [inline scalding-serialization-macros/src/test/scala/com/twitter/scalding/serialization/macros/MacroOrderingProperties.scala]: Were these changes from your GUI or the formatting script?

Comment (oeddyo) [inline scalding-serialization-macros/src/test/scala/com/twitter/scalding/serialization/macros/MacroOrderingProperties.scala]: That's strange. I didn't notice the change. 

That seems to be changed when I ran 

./sbt test

Comment (johnynek) [inline scalding-serialization-macros/src/test/scala/com/twitter/scalding/serialization/macros/MacroOrderingProperties.scala]: probably from scalariform.

**Pull request merged**